src/app/modules/submission/multirespondent-submission/multirespondent-submission.service.ts

```diff
@@ -674,25 +676,36 @@ export const performMultiRespondentPostSubmissionCreateActions = ({
   }
 
   const webhookUrl = form.webhook?.url
+
   if (webhookUrl) {
     logger.info({
-      message: 'Sending initial webhook for multirespondent submission',
+      message: 'Sending update webhook for multirespondent submission',
```

Comment:
question: wondering why this might have been changed to `update`? am a little confused on what this change means since the else statement below invoked is called `sendInitialWebhook`

Reply:
what is the difference between `initial` and `update`?